lib/presentation/screens/general_settings_screen/general_settings_layout.dart

```diff
@@ -0,0 +1,140 @@
+import 'package:easy_localization/easy_localization.dart';
```

Comment:
Could not find a generator for route RouteSettings("general_settings_screen", null) in the _WidgetsAppState.